src/core/thread/mle.hpp

```diff
@@ -168,10 +163,9 @@ class Mle : public InstanceLocator, private NonCopyable
      * This method generates an MLE Announce message.
      *
      * @param[in]  aChannel        The channel to use when transmitting.
```

Comment:
Is there a standard about the number of spaces between the parameter and the comment? There are 8 spaces between the `aChannel` and the comment, are there too many spaces?

Reply:
I don't think we have any specific rule/policy on this. Make sense to me to leave it to developer.